### DISCUSSION TITLE
Customized admonitions

### DISCUSSION THREAD

**Discussion by spinningrachel** [squidfunk/mkdocs-material#2636] [Q&A]
Hi again. I created a couple of customized admonitions. They are all rendering correctly when I run mkdocs serve from my local environment, but after gh-deploy - not the case ... 

One example is this: https://docs.lightrun.com/agentadmin-agentprops/

This page has the new "imp" admonition. 

All of the customizations are configured in lightrun.css
The repo is accessible directly from the docs. 

Any suggestions?

**Comment by Andre601** [comment]
First of all: What should be shown vs. what is shown?
